client/thelounge.webmanifest

```diff
@@ -6,8 +6,7 @@
 	"display": "standalone",
```

Comment:
Should this file use spaces as well like our other JSON files, or do we want to keep tabs here? (My guess is the latter, I don't mind either way)